test/parallel/test-http-should-keep-alive.js

```diff
@@ -20,10 +20,11 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+-require('../common');
```

Comment:
Almost all good but it looks like there's a `-` sign ahead of the require statement.